### ISSUE TITLE
Ivis produces different embeddings from a saved model across multiple tensorflow sessions

### ISSUE THREAD

**Issue by idroz** [beringresearch/ivis#82]
Running `ivis.transform` on a pre-built model across different tensorflow sessions produces different embeddings. Embeddings are consistent within a session, but once tf session is restarted and model is reloaded, embeddings change.

This was introduced in 2.0 upgrade as earlier versions of ivis behaved as expected. Additionally, this only seems to affect larger datasets. I don't see this in iris dataset, but in 500k+ row dataset it's present.

Things I checked that seem to be ok:

**model loading**: model weights, optimizer weights all appear to be consistent between sessions. So this isn't an issue with incorrect initialisation

**toggling GPU training**: the bug seems to be present when running both CPU and GPU transformations.

**data normalization**: data normalization stays consistent i.e. input data is not altered between sessions.

**Comment by idroz** [comment]
Unrelated to ivis :)
Closing
